### ISSUE TITLE
Icons should have hover tooltips & aria labels for accessibility

### ISSUE THREAD

**Issue by ZachTRice** [nasa-gibs/worldview#2564]
**What existing feature could be improved?**
The share, projection, camera, info and measurement tools do not describe what they are. Having a tooltip describe the button on hover would be helpful to new users and those that are impaired. Additionally, we should add aria-labels to these buttons for enhanced accessibility.

**Describe the improvement you'd like**
- Tooltips on icon hover
- Aria labels on icons/buttons

**Describe alternatives you've considered**
None.

**Additional context**
N/A

**Comment by ZachTRice** [comment]
After looking at this again, it appears we do have tooltips but the delay before they show up is very long. We should make them appear a bit sooner on hover

**Comment by minniewong** [comment]
We could use the Reactstrap tooltip

**Comment by ZachTRice** [comment]
That would work
